internal/rbd/mirror.go

```diff
@@ -33,6 +33,39 @@ func (ri *rbdImage) EnableImageMirroring(mode librbd.ImageMirrorMode) error {
 	}
 	defer image.Close()
 
+	parentInfo, err := image.GetParent()
+	if err != nil {
+		return fmt.Errorf("failed to get parent info of image %q: %w",
+			ri, err)
+	}
+
+	if parentInfo.Image.Trash {
+		return fmt.Errorf("failed to enable mirroring on image %q:"+
+			" parent %q is in trash",
+			parentInfo.Image.ImageID, ri)
```

Comment:
returning ImageID is not useful, if possible return parent name or just discard parent id or log `ImageName`